components/tools/OmeroPy/src/omero/cli.py

```diff
@@ -1567,6 +1568,10 @@ def __call__(self, arg):
             for id in parts[1].split(","):
                 if "-" in id:
                     needsForce = True
+                    warnings.warn("Using '--dry-run'. \
```

Comment:
I get it in all 3 cases:

## no arg
```
[jamoore@ome-c6100-1 dist]$ bin/omero delete Image:888888888-999999999
/opt/hudson/workspace/OMERO-DEV-merge-deploy/src/dist/lib/python/omero/cli.py:1580: DeprecationWarning: Using '--dry-run'.                                   Future versions will switch to '--force'.                                   Explicitly set the parameter for                                   portability
  portability", DeprecationWarning)
```

## with `--dry-run`
```
[jamoore@ome-c6100-1 dist]$ bin/omero delete Image:888888888-999999999 --dry-run
/opt/hudson/workspace/OMERO-DEV-merge-deploy/src/dist/lib/python/omero/cli.py:1580: DeprecationWarning: Using '--dry-run'.                                   Future versions will switch to '--force'.                                   Explicitly set the parameter for                                   portability
  portability", DeprecationWarning)
```

## with `--force`
```
[jamoore@ome-c6100-1 dist]$ bin/omero delete Image:888888888-999999999 --force
/opt/hudson/workspace/OMERO-DEV-merge-deploy/src/dist/lib/python/omero/cli.py:1580: DeprecationWarning: Using '--dry-run'.                                   Future versions will switch to '--force'.                                   Explicitly set the parameter for                                   portability
  portability", DeprecationWarning)
```